index.html

```diff
@@ -367,45 +368,64 @@ <h3>
           return [=a promise rejected with=] {{"NotAllowedError"}}
           {{DOMException}}.
           </li>
```

Comment:
Nit: steps 1.1 and 1.2 should be steps 2 and 3, i.e. they should not be indented.

Reply:
Done in #308.